miden/tests/integration/stdlib/math/ext5_scalar.rs

```diff
@@ -0,0 +1,359 @@
+use super::build_test;
+use std::cmp::PartialEq;
+use std::ops::Mul;
```

Comment:
minor nit: I'd combine these into a single import line.

Reply:
Missed it, fixed in a7660d706e3c70f12196aceb5fb1fab83efd4475